src/hotspot/share/oops/klassVtable.cpp

```diff
@@ -494,6 +502,11 @@ bool klassVtable::update_inherited_vtable(const methodHandle& target_method,
           allocate_new = false;
         }
 
+        // Set the vtable index before the constraint check safepoint potentially
+        // redefines this method, which is possible if it is a default method belonging
+        // to a super class or interface.
```

Comment:
Minor nit, wording is awkward.  Maybe:
Set the vtable index before the constraint check safepoint, which potentially redefines this method if this method is a default method belonging to a super class or interface.

Reply:
The suggested rewrite reads better.

Reply:
ok, that wording is more clear. thank you!